settings.gradle.kts

```diff
@@ -6,7 +6,6 @@ pluginManagement {
     }
 }
 dependencyResolutionManagement {
-    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
```

Comment:
adding the JS/WASM targets requires this, as they add the NPM repo under the hood.